debian/jitsi-meet-tokens.postinst

```diff
@@ -58,6 +58,7 @@ case "$1" in
                 sed -i "s/ --app_id=\"example_app_id\"/ app_id=\"$APP_ID\"/g" $PROSODY_HOST_CONFIG
                 sed -i "s/ --app_secret=\"example_app_secret\"/ app_secret=\"$APP_SECRET\"/g" $PROSODY_HOST_CONFIG
                 sed -i 's/ --modules_enabled = { "token_verification" }/ modules_enabled = { "token_verification" }/g' $PROSODY_HOST_CONFIG
```

Comment:
As this is no longer used, can you remove it, please?

Reply:
At first I thought to delete it too but I gave up because it might be someone whose the prosody config is old (which is containing this line) and might decide to install `jitsi-meet-tokens` one day.

what is your suggestion?

Reply:
Well ... let me check when we changed that

Reply:
Well, it was done 9-10 months ago ... I would say let's clean it, on the other hand, it does not bother anybody leaving it there. At least we had discussed it ... and someone can see the discussion in 10 months and delete it :P merging it.
https://github.com/jitsi/jitsi-meet/commit/db6a2673dec63210de3d2b2baf1279e8dec46b63